.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
       EVENT_TYPE='push pull_request cron'
 
   global:
+    - SETUPTOOLS_VERSION=27
```

Comment:
strictly speaking we could use a maximum version number, but I'm fine with pinning it to be this precise.

Reply:
Good point - if you are happy leaving it as is, let's do that, since it's (hopefully) only temporary.

Reply:
sure, I suspect it's available for all supported python versions (for the backport), if not we can always change it there.